pallets/lbp/src/tests.rs

```diff
@@ -1361,6 +1365,190 @@ fn remove_liquidity_should_work() {
 	});
 }
 
+#[test]
+fn remove_liquidity_and_create_xyk_should_work() {
+	predefined_test_ext().execute_with(|| {
+		// create a pool that ends with 50/50 weights
+		assert_ok!(LBPPallet::create_pool(
+			Origin::root(),
+			ALICE,
+			HDX,
+			1_000_000_000,
+			BSX,
+			2_000_000_000,
+			20_000_000,
+			50_000_000,
+			WeightCurveType::Linear,
+			DEFAULT_FEE,
+			CHARLIE,
+			0,
+		));
+
+		assert_ok!(LBPPallet::update_pool_data(
+			Origin::signed(ALICE),
+			HDX_BSX_POOL_ID,
+			None,
+			SALE_START,
+			SALE_END,
+			None,
+			None,
+			None,
+			None,
+			None,
+		));
+
+		set_block_number::<Test>(41);
+
+		assert_err!(
```

Comment:
```suggestion
		assert_noop!(
```
should be noop, right?